tools/AzureRM.BootStrapper/AzureRM.BootStrapper.Tests.ps1

```diff
@@ -12,7 +12,7 @@ Describe "Get-ProfileCachePath" {
                 $result = Get-ProfileCachePath
                 $result | Should Match "(.*)ProfileCache$"
                 $result.Contains("ProgramData") | Should Be $true
-                Assert-VerifiableMocks
+                Assert-VerifiableMock
```

Comment:
Renaming Assert-VerifiableMocks to Assert-VerifiableMock was done because the build was complaining about it.